src/js/redux/reducers/search/searchFiltersReducer.js

```diff
@@ -19,7 +19,14 @@ import * as ProgramSourceFilterFunctions from './filters/programSourceFilterFunc
 // frontend will reject inbound hashed search filter sets with different versions because the
 // data structures may have changed
 
-export const filterStoreVersion = '2019-07-26';
+// export const filterStoreVersion = '2019-07-26';
+export const filterStoreVersion = '2017-11-21';
```

Comment:
@ebdabbs do we need to change this? I changed it to match what was being returned by the mock server.

Reply:
Yes, please! Since the filter object has changed, old search hashes will no longer work correctly after this is merged.